pylintrc

```diff
@@ -10,7 +10,7 @@
 # Profiled execution.
 profile=no
 
-# Add <file or directory> to the black list. It should be a base name, not a
```

Comment:
Or maybe simply "ignore <file or directory>". An alternative can be to check what more recent pylint versions have for these keys in the configuration file.

Reply:
Current documentation uses `to be skipped`, which will fit here just fine.